terraform/stacks/umccr_data_portal/workflow/wts_tumor_only.tf

```diff
@@ -7,7 +7,7 @@ locals {
 
   wts_tumor_only_wfl_version = {
     dev  = "3.9.3"
-    prod = "3.9.3--e746073"
+    prod = "3.9.3--a0960f3"
```

Comment:
I don't think this version ID is correct, PR https://github.com/umccr/cwl-ica/pull/284 created the production version (3.9.3--8fdabfc) for the dragen transcriptome pipeline (see https://github.com/umccr/cwl-ica/pull/284/files#diff-b47734da5a8e4c08b51d28d2b39f874c872377c258d5fecc1d5072c93b2572dfR1805-R1809) for more info. 

Can also run the following to validate
```
ica workflows versions list wfl.7e5ba7470b5549a6b4bf6d95daaa1214 -o json | \
jq --raw-output \
  '
    .items | 
    sort_by(.timeModified) | 
    reverse | 
    map(.version)[0:5]
  '
```

Yields

```
[
  "3.9.3--8fdabfc",
  "3.7.5--8fdabfc",
  "3.9.3--a0960f3",
  "3.9.3--a8fdcf1",
  "3.9.3--3b07571"
]
```

Reply:
Correction: this does contain https://github.com/umccr/cwl-ica/pull/281 but I'd still update to https://github.com/umccr/cwl-ica/pull/284 to ensure if we do want to move to using presigned urls at some point.

Reply:
Okay, as noted I had pointed to https://github.com/umccr/cwl-ica/pull/281. Now updated to include https://github.com/umccr/cwl-ica/pull/284 in https://github.com/umccr/infrastructure/commit/fe2a41e8aca050fe8a2eb637346ee79ce41cc70f